### PR TITLE
Create ORCA release 9.0.1 containing documentation and security updates

### PR DESCRIPTION
**Pull request by rizbihassan**
## Summary of Changes

Addresses [ORCA-768: Create ORCA release 9.0.1 containing documentation and security updates](https://bugs.earthdata.nasa.gov/browse/ORCA-768)

## Changes

* Detailed list or prose of changes

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?

Deployed in AWS.

## Checklist:

- [x ] My code follows the style guidelines of this project
- [ x] I have performed a self-review of my own code
- [x ] I have commented my code, particularly in hard-to-understand areas
- [x ] I have made corresponding changes to the necessary documentation (remove those that do not apply)
    - [ x] CHANGELOG.md
- [x ] My code has passed security scanning
    - [x ] Snyk
    - [ x] git-secrets

### CODE REVIEW

**Review by zroehrich — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -15,10 +15,27 @@ and includes an additional section for migration notes.
 
 ## [Unreleased]
```
Comment (bhazuka) [inline CHANGELOG.md:16]: Add the following sections under Unreleased so it is available for the next release

### Added
### Changed
### Deprecated
### Removed
### Fixed
### Security
### Migration Notes

File: CHANGELOG.md [hunk 2]
```diff
@@ -15,10 +15,27 @@ and includes an additional section for migration notes.
...
+
+- *ORCA-737* Updated moto library used for unit tests to version 4.2.2 from version 2.0.
+
+- *ORCA-731* Updated boto3 library used for unit tests to version 1.28.76 from version 1.18.40 to fix unit test warnings.
+
+- *ORCA-704* Updated dr-buckets.tf.template and buckets.tf.template with provider block to deploy in the us-west-2 region due to deployments failing in the other regions.
+
+- *ORCA-708* Updated integration_test/shared/setup-orca.sh script to use the root folder instead of cloning in a duplicate repository.
+
 ### Deprecated
```
Comment (bhazuka) [inline CHANGELOG.md:39]: Delete since we have no Deprecated items this release

File: CHANGELOG.md [hunk 3]
```diff
@@ -15,10 +15,27 @@ and includes an additional section for migration notes.
...
 
 ### Changed
 
+- *ORCA-780* Updated ORCA "Deployment with Cumulus" documentation with instructions and examples to run ORCA recovery and archive workflows.
+
+- *ORCA-778* Upgraded Docusaurus to version 2.4.3 to fix snyk vulnerabilities and security issues.
+
+- *ORCA-765* Updated ORCA "Creating the Glacier Bucket" documentation with instructions to deploy ORCA DR buckets using cloudformation.
+
+- *ORCA-737* Updated moto library used for unit tests to version 4.2.2 from version 2.0.
```
Comment (bhazuka) [inline CHANGELOG.md:31]: Move this line `ORCA-737` under security

**Review by bhazuka — CHANGES_REQUESTED** [review]
Some minor items in the Changelog to shuffle and add.

File: CHANGELOG.md [hunk 4]
```diff
@@ -15,10 +15,27 @@ and includes an additional section for migration notes.
...
 ### Added
+- *ORCA-766* Created AWS cloudformation template that can be used to deploy ORCA DR buckets.
 
 ### Changed
 
+- *ORCA-780* Updated ORCA "Deployment with Cumulus" documentation with instructions and examples to run ORCA recovery and archive workflows.
+
+- *ORCA-778* Upgraded Docusaurus to version 2.4.3 to fix snyk vulnerabilities and security issues.
+
+- *ORCA-765* Updated ORCA "Creating the Glacier Bucket" documentation with instructions to deploy ORCA DR buckets using cloudformation.
```
[the remark this answers is not in the document]
Comment (rizbihassan) [reply CHANGELOG.md]: Moved.

File: CHANGELOG.md [hunk 5]
```diff
@@ -15,10 +15,27 @@ and includes an additional section for migration notes.
...
+
+- *ORCA-737* Updated moto library used for unit tests to version 4.2.2 from version 2.0.
+
+- *ORCA-731* Updated boto3 library used for unit tests to version 1.28.76 from version 1.18.40 to fix unit test warnings.
```
[the remark this answers is not in the document]
Comment (rizbihassan) [reply CHANGELOG.md]: moved

File: CHANGELOG.md [hunk 6]
```diff
@@ -15,10 +15,27 @@ and includes an additional section for migration notes.
...
+
+- *ORCA-704* Updated dr-buckets.tf.template and buckets.tf.template with provider block to deploy in the us-west-2 region due to deployments failing in the other regions.
+
+- *ORCA-708* Updated integration_test/shared/setup-orca.sh script to use the root folder instead of cloning in a duplicate repository.
+
 ### Deprecated
 
 ### Removed
```
[the remark this answers is not in the document]
Comment (rizbihassan) [reply CHANGELOG.md]: Removed.

File: CHANGELOG.md (hunk 3 above)
Comment (rizbihassan) [reply CHANGELOG.md]: Moved.

File: CHANGELOG.md [hunk 7]
```diff
@@ -15,10 +15,27 @@ and includes an additional section for migration notes.
...
+## [9.0.1] 2023-11-16
+
 ### Added
+- *ORCA-766* Created AWS cloudformation template that can be used to deploy ORCA DR buckets.
 
 ### Changed
 
+- *ORCA-780* Updated ORCA "Deployment with Cumulus" documentation with instructions and examples to run ORCA recovery and archive workflows.
+
+- *ORCA-778* Upgraded Docusaurus to version 2.4.3 to fix snyk vulnerabilities and security issues.
```
[the remark this answers is not in the document]
Comment (rizbihassan) [reply CHANGELOG.md]: Moved.

File: CHANGELOG.md (hunk 2 above)
Comment (rizbihassan) [reply CHANGELOG.md]: Removed.

File: CHANGELOG.md (hunk 1 above)
Comment (rizbihassan) [reply CHANGELOG.md:16]: Added.

File: CHANGELOG.md [hunk 8]
```diff
@@ -15,35 +15,35 @@ and includes an additional section for migration notes.
 
 ## [Unreleased]
 
+- *Migration Notes* - Notes for end users to migrate to the version.
```
Comment (bhazuka) [inline CHANGELOG.md:18]: Should be headdings L3 for all of these
like the following
`- *Migration Notes* - Notes for end users to migrate to the version.` becomes
`### Migration Notes`

**Review by bhazuka — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 8 above)
Comment (rizbihassan) [reply CHANGELOG.md]: Fixed.

**Review by bhazuka — APPROVED** [review]

**Pull request merged**